source/reference/minio-mc-admin/mc-admin-accesskey-create.rst

```diff
@@ -0,0 +1,232 @@
+.. _minio-mc-admin-accesskey-create:
+
+=============================
+``mc admin accesskey create``
+=============================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. mc:: mc admin accesskey create
+
+
+Syntax
+------
+
+.. start-mc-admin-accesskey-create-desc
+
+The :mc-cmd:`mc admin accesskey create` command adds a new access key and secret key pair for an existing MinIO user.
+
+.. end-mc-admin-accesskey-create-desc
+
+.. admonition:: Access keys for OpenID Connect or AD/LDAP users
+   :class: note
+
+   This command is for access keys for users created directly on the MinIO deployment and not managed by a third party solution.
+
+   - To generate access keys for :ref:`OpenID Connect users <minio-external-identity-management-openid>`, use the :ref:`MinIO Console <minio-console>`.
```

Comment:
Related topic: will this functionality continue to be in the MinIO Console? I've lost track.

Reply:
The MinIO Console isn't going anywhere. As far as I understand, yes, this continues to be there.